src/custom-elements/docs/docs.json

```diff
@@ -2159,6 +2159,22 @@
       "docsTags": [],
       "usage": {},
       "props": [
+        {
+          "name": "decimals",
```

Comment:
Can we avoid the prop? The decimals format should be consistent with the Step, right? if the step is set to 0.01, we could set the format to X.00 under the hood, no need for another prop to achieve the same.
@jllr @dani-cl-madrid Thoughts?

Reply:
Sounds good to be consistent with the number of decimals from the step. @janibolkvadze @josecarlosgonzalezv